powers.c

```diff
@@ -0,0 +1,28 @@
+/* Albert Zhang
+   Prints powers of 2 up to the user's input */
```

Comment:
you misunderstood the assignment - n isn't the number being input. The number being input is the number that 2^n should be less than. you also forgot 2^0 = 1.